### DISCUSSION TITLE
Why does Civitai look like an anime porn site ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1252] [General]
I know that porn has always driven tech, but this is ridiculous ! 

Is anyone using Stable Diffusion for anything other than waifu production ?

Thanks !

**Comment by vladmandic** [comment]
and why ask that question here? lol.
besides, plenty of models are on huggingface, you don't need to go to civit.ai.

**Reply by swamp-cabbage** [reply]
💬GeneralDiscussions
Chat about anything and everything here !

**Reply by vladmandic** [reply]
sure, go for it, i'm not locking or deleting the thread as long as there is no abuse :)

**Comment by brknsoul** [comment]
https://civitai.com/tag/photorealistic
Problem solved.
